packages/integration/src/ScmIntegrations.test.ts

```diff
@@ -73,4 +73,16 @@ describe('ScmIntegrations', () => {
     expect(i.byHost('github.local')).toBe(github);
     expect(i.byHost('gitlab.local')).toBe(gitlab);
   });
+
+  it('can resolveUrl using fallback', () => {
```

Comment:
Think we should make this work as one might expect?

```ts
i.resolveUrl('./catalog-info.yaml', 'https://github.com/backstage/backstage/tree/master')
```

Reply:
Hm to get that working we should override this method in the github integration. Separate PR?

Reply:
Yeah could do it separately if you want, I think we can say that the current behaviour is borked enough that I don't think we can expect any broad usage that would rely on it